lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -190,6 +190,16 @@ class LighthouseReportViewer {
 
       const features = new ViewerUIFeatures(dom, saveCallback);
       features.initFeatures(json);
+      features.initSwapLocale({
+        async fetchData(localModuleName) {
```

Comment:
`localModuleName` is roughly a `canonicalLocaleName` right?

Reply:
I'm thinking yes, but I'm finding that part of the locale code hard to understand. if so then yea some better names are in order.